### ISSUE TITLE
Resccue Data Package

### ISSUE THREAD

**Issue by ghilbrae** [clarity-h2020/data-package#57]
It has been agreed that results from the project Resccue will be added to the CSIS. 

Meteogrid will help them with this task.

**Comment by ghilbrae** [comment]
We have received a bunch of layers from the Resccue team We are still working on them to upload them to our geoserver but a preliminary result can be viewed at: https://clarity.meteogrid.com/geoserver/resccue/wms?service=WMS&version=1.1.0&request=GetMap&layers=resccue%3ABCN_pr_100years_period2011-2040_cuantil10_3035&bbox=3514242.2563050915%2C1957497.0370172886%2C3786106.8423613026%2C2204505.4323483603&width=768&height=697&srs=EPSG%3A3035&format=application/openlayers#toggle

**Comment by ghilbrae** [comment]
The layers pertaining to Barcelona have been all uploaded to geoserver and can be accessed as usual. A list of all of them can be seen in the Layer Preview filtering by "resccue".

**Comment by DenoBeno** [comment]
Time to start working on a new data package?

I would advise to just put one entry for each type of the resources they have, so that the data package exists, something can be viewed and we are ready for finalising it once the resources can be defined with variables.

**Comment by p-a-s-c-a-l** [comment]
> https://clarity.meteogrid.com/geoserver/resccue/wms?service=WMS&version=1.1.0&request=GetMap&layers=resccue%3ABCN_pr_100years_period2011-2040_cuantil10_3035&bbox=3514242.2563050915%2C1957497.0370172886%2C3786106.8423613026%2C2204505.4323483603&width=768&height=697&srs=EPSG%3A3035&format=application/openlayers#toggle

```xml
<ServiceExceptionReport version="1.1.1" >   
        <ServiceException code="LayerNotDefined" locator="layers">
                 Could not find layer resccue:BCN_pr_100years_period2011-2040_cuantil10_3035
        </ServiceException>
</ServiceExceptionReport>
```

Is there already something else available on GeoServer or in CSIS? The question is also, what is the purpose of this RESCCUE data package? For which kind of study can it be used? Or resources to other data packages, e.g. the [European Wide Data Package](https://csis.myclimateservice.eu/node/682)?

BTW, "variables" (= **Template Resources**) will only work in screening studies or such studies with "presets" (= **Scenarios**) enabled as explained [here](https://github.com/clarity-h2020/data-package/issues/47#issuecomment-557562073).

**Comment by ghilbrae** [comment]
That layer no  longer exists, I changed some information to get a better classification of the layers. You can try this link instead:

https://clarity.meteogrid.com/geoserver/resccue/wms?service=WMS&version=1.1.0&request=GetMap&layers=resccue%3ABCN_climatic_pr_100years_period2011-2040_cuantil10&bbox=3514242.2563050915%2C1957497.0370172886%2C3786106.8423613026%2C2204505.4323483603&width=768&height=697&srs=EPSG%3A3035&format=application/openlayers

> Is there already something else available on GeoServer or in CSIS? The question is also, what is the is the purpose of this RESCCUE data package? For which kind of study can it be used? Or resources to other data packages, e.g. the [European Wide Data Package](https://csis.myclimateservice.eu/node/682)?

There are around 125 layers related to RESCCUE in our geoserver right now, all from the study for Barcelona. As far as I know there is still more data that needs to be uploaded. 

I'm working on creating resources for some of the layers in the CSIS and a DP.

Regarding the usefulness of this, I think @luis-meteogrid is better suited to give you an answer than I am.

> 
> BTW, "variables" (= **Template Resources**) will only work in screening studies or such studies with "presets" (= **Scenarios**) enabled as explained [here](https://github.com/clarity-h2020/data-package/issues/47#issuecomment-557562073).

Thanks! I'll take a look at this and change whatever is needed in what I've already done to fit that.

**Comment by DenoBeno** [comment]
Keep our templates in mind when naming the layers. It will be easier to
make that work if the same naming conventions (rcp, period, frequency,
yell if more is needed) are used everywhere.

Angela Rivera <notifications@github.com> schrieb am Di., 10. Dez. 2019,
09:38:

**Comment by p-a-s-c-a-l** [comment]
See also https://github.com/clarity-h2020/local-effects/issues/9#issuecomment-564022191 for the current list of supported variables in template resources.

**Comment by p-a-s-c-a-l** [comment]
Any progress to report here?

**FYI:** I've create a [wiki page](https://github.com/clarity-h2020/data-package/wiki/Template-Resources) that explains how to use **Template Resources** in [Data Packages](https://github.com/clarity-h2020/data-package/wiki/Creating-a-Data-Package). An example can be found [here](https://github.com/clarity-h2020/data-package/issues/21#issuecomment-587473641).

**Comment by ghilbrae** [comment]
TBH, there's not much to report we have only received a handful of new data, most of it in PDF or PNG (!) and all of it is private so very difficult to add to the DP. 

I'll keep it open for a little longer and then review the status.

**Comment by p-a-s-c-a-l** [comment]
FYI: https://github.com/clarity-h2020/data-package/issues/21#issuecomment-587473641

**Comment by p-a-s-c-a-l** [comment]
Are there still plans to make this data available in CSIS?

If not, do we need to keep the meta-data in CKAN and thus also in the Data Management Plan?
There are several issues with [these datasets](https://ckan.myclimateservice.eu/dataset/?tags=RESCCUE), that would have to be resolved before we can publish [the DMP](https://github.com/clarity-h2020/ckan/issues/32). Some RESCCUE datasets are marked as **private** which is not allowed in the DMP (**meta**-data must be public).

**Comment by p-a-s-c-a-l** [comment]
Is this still being worked on? Otherwise we should close this issue.

**Comment by p-a-s-c-a-l** [comment]
[Data Package in CSIS](https://csis-dev.myclimateservice.eu/node/2198)

**Comment by p-a-s-c-a-l** [comment]
> Is this still being worked on? Otherwise we should close this issue.

**Comment by ghilbrae** [comment]
We are still adding new layers to the Data Package. The people from the RESCCUE project have recently sent more information our way.

Anyway, we can close the issue and open a new one if there is some sort of problem or question.
